### PR TITLE
Tweak research language to clarify percentage multiplier

### PR DESCRIPTION
**Pull request by arinthros**

### CODE REVIEW

File: myning/research.yaml [hunk 1]
```diff
@@ -37,17 +37,16 @@ species_discovery:
...
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
```
Comment (arinthros) [inline myning/research.yaml:50]: This one had 11 description options, so I removed the last unused one.

File: myning/research.yaml [hunk 2]
```diff
@@ -37,17 +37,16 @@ species_discovery:
...
+    - Increase the likelihood of recruiting more rare species by 10%
+    - Increase the likelihood of recruiting more rare species by 20%
+    - Increase the likelihood of recruiting more rare species by 30%
+    - Increase the likelihood of recruiting more rare species by 40%
+    - Increase the likelihood of recruiting more rare species by 50%
+    - Increase the likelihood of recruiting more rare species by 60%
+    - Increase the likelihood of recruiting more rare species by 70%
+    - Increase the likelihood of recruiting more rare species by 80%
+    - Increase the likelihood of recruiting more rare species by 90%
+    - Increase the likelihood of recruiting more rare species by 100%
```
Comment (TheRedPanda17) [inline myning/research.yaml:49]: This is not related to percentage at all. It needs to change back.

Comment (arinthros) [reply myning/research.yaml:49]: What is it related to? There are 10, 20, etc. values but the descriptions don't reflect those value changes.

Comment (arinthros) [reply myning/research.yaml:49]: Is it the common, uncommon, etc?

Comment (arinthros) [reply myning/research.yaml:49]: I'm new to python, but I think this is the line that does the math.
```py
weight + RESEARCH["species_rarity"].player_value + (i * 3)
```

Comment (TheRedPanda17) [reply myning/research.yaml:49]: It is related to the tier, but the increases are not neat percentages

**Review by TheRedPanda17 — APPROVED** [review]
